napari/util/app.py

```diff
@@ -0,0 +1,10 @@
+import sys
+from contextlib import contextmanager
+from PyQt5.QtWidgets import QApplication
+
+
+@contextmanager
+def app_context():
```

Comment:
Would rather see a wrapper of `QApplication` (basically only needs to include the `exec` method) that also implements `__enter__` and `__exit__`.

Reply:
@kne42 I much prefer this style of context manager definition, as it's easy to see all the code (entry, yield, exit) together. I think it's a style issue only (?), and if so then I'd rather keep it like this than add to the already cumbersome Qt class hierarchy.